src/app/zwiazek/kalendarz/_components/CalendarTableCell.tsx

```diff
@@ -0,0 +1,44 @@
+import { formatDate } from "@/utils";
+import clsx from "clsx";
+
+interface Props {
+  day: string;
+  dayIndex: number;
+  currentDate: Date;
+  eventsDates: string[];
+}
+
+export const CalendarTableCell = ({
+  day,
+  dayIndex,
+  currentDate,
+  eventsDates,
+}: Props) => {
+  const today = formatDate(new Date().toString());
+  const dayNumber = day.split(".")[0];
+  const monthNumber = day.split(".")[1];
+  const isToday = day === today;
+
+  const isCurrentMonth =
+    monthNumber === formatDate(currentDate.toString()).split(".")[1];
```

Comment:
Just use month getter and parse it to string (please remember it is indexed from `0`)